README.md

```diff
@@ -45,20 +58,48 @@ Password: Dynamic33
 
 5. **Run the Analysis Code**
 
-In RStudio, open `vancouver-tree-height-geo.qmd` and run all code chunks.
+open a terminal (from the docker rstudio) and run the following commands:
+
+```
+Rscript scripts/00_download_data.R "https://opendata.vancouver.ca/api/explore\
```

Comment:
There should be a space before each backslash char `\`, for readability (as-is, it kind of looks like they are part of the command. Adding spaces will help)